dotCMS/src/main/java/com/dotcms/rest/api/v1/content/ContentVersionResource.java

```diff
@@ -165,6 +172,44 @@ public Response findVersions(@Context final HttpServletRequest request,
 
     }
 
+    @GET
```

Comment:
missing swagger and javadoc

Reply:
Yes, that's all part of the new standar we have for REST Endpoints and will be added soon.